bootstrap/039_reference_management.sql

```diff
@@ -25,6 +26,5 @@ begin
     else
        return 'Unknown operation: ' || operation;
   end case;
-  return 'Success';
```

Comment:
this lin elooks weird. Why are we getting rid of information?

Reply:
We have a mixed bag of when it comes to what does it mean when a procedure returns a string. Sometimes it has a sentinel string like "Success" and sometimes it is an error message (when we detect something specific was wrong).

My goal was to make this procedure consistent with the other ones in this PR so we have fewer special cases from the caller's POV. If you'd rather not continue that convention, that's also fine (raising exceptions is probably what we should have been doing all along).